app/Models/Traits/Scoreable.php

```diff
@@ -74,7 +74,7 @@ public function accuracy(): float
 
     public function recalculateRank(): void
     {
-        if (!$this->passed) {
+        if (!$this->pass) {
```

Comment:
`Solo\Score` doesn't have a `pass`; actually, the function should probably be moved out of the trait since it's only used by legacy scores.

Reply:
but Solo\Score doesn't use this trait? 🤔

Reply:
does it not? 🤔

Reply:
Oh wait, I mean because the naming in `Solo\Score::makeLegacyEntry` with the variable being `$score` and the method being named `recalculateRank` makes it confusing and the method is only used in legacy scores now.

I guess that can be a separate issue 🤷